guide/english/java/arrays/index.md

```diff
@@ -50,6 +50,9 @@ double[] List = new double[10];
 ```java
 dataType[] arrayName = {value_0, value_1, ..., value_k};
 ```
+```java
+dataType[] arrayName = new dataType[] {value_0, value_1, ..., value_k};
```

Comment:
The "Another way to create an Array" section shows the creation of array using the literal `{ }` notation. Showing another example using the `new` keyword is not required here. Please delete this example.